docs/2-attack-of-the-pipelines/3b-tekton.md

```diff
@@ -87,7 +87,7 @@ The task takes parameter needed to build and push the image as parameters, such
 
 ![image-push-script](./images/image-push-script.png)
 
-#### 4 - stakater-helm-push-v1 🅿
+#### 4 - `stakater-helm-push-v1` 🅿
```

Comment:
not this one, not sure why its being set like this, it is not consistent with the  rest of items in this list

Reply:
fixed other headings